worker/worker.go

```diff
@@ -1257,7 +1179,6 @@ func (w *Worker) Handler() http.Handler {
 
 		"GET    /rhp/contracts":              w.rhpContractsHandlerGET,
 		"POST   /rhp/contract/:id/broadcast": w.rhpBroadcastHandler,
-		"POST   /rhp/contract/:id/prune":     w.rhpPruneContractHandlerPOST,
 		"GET    /rhp/contract/:id/roots":     w.rhpContractRootsHandlerGET,
```

Comment:
I'd f/u with a removal of this endpoint as well. To get rid of everything contract related from the worker eventually.

As I understand this is only used in testing right now so we probably need to convert those tests to directly ask the host for the roots of the contract. e.g. via `cluster.hosts[0].store.SectorRoots()`

Reply:
I'll just do it here it's not that much